src/App.jsx

```diff
@@ -14,30 +15,78 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
+  const [message, setMessage] = useState(`${selectedGood} is selected`);
```

Comment:
The initial state of `message` is set using the `selectedGood` state. If the initial value of `selectedGood` changes, the `message` will not automatically update to reflect this change. Consider using a `useEffect` hook to update `message` whenever `selectedGood` changes, ensuring consistency between the two states.